projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/jobData.ts

```diff
@@ -48,3 +49,19 @@ export function getJobDataJsonObject() {
   };
   return jsObj;
 }
+
+/*
+ * Function that checks whether a value is defined in jobData
+ * Shows dialog that operation cannot be performed because of undefined value
+ */
+export async function checkDefinedValue(option: VdkOption): Promise<boolean> {
+  if (jobData.get(option)) return true;
```

Comment:
nit: What is the expectation, having a value by key? Or having a key with a potentially empty value is also correct?

Reply:
Here we check whether the value is "" or not.

Reply:
what's the expectation, logically?

Reply:
a string that is not empty the other test are done in the server extension
But the logic is up to be changed with getting the props dynamically from the config.ini